src/style.css

```diff
@@ -0,0 +1,28 @@
+
+body {
+  margin: 0;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  margin-right: 4px;
+  background-image: url(images/star.svg);
+  background-repeat: no-repeat;
+  background-position: 50%;
+}
+
+.stars--1 > :first-child,
+.stars--2 > :nth-child(-n + 2),
+.stars--3 > :nth-child(-n + 3),
+.stars--4 > :nth-child(-n + 4),
+.stars--5 > :nth-child(-n + 5) {
+  height: 16px;
+  width: 16px;
+  background-image: url(images/star-active.svg);
+  background-repeat: no-repeat;
```

Comment:
you have all these styles except  changed background-image in `.stars__star` - so do not repeat them